src/main/kotlin/net/horizonsend/ion/Ion.kt

```diff
@@ -83,9 +83,19 @@ class Ion : JavaPlugin() {
 			Material.RED_WOOL,
 			Material.BLACK_WOOL
 		).forEach {
-			val woolType = ShapelessRecipe(NamespacedKey(this, "${it.name.lowercase()}_string_recipe"), ItemStack(Material.STRING, 4))
+			val woolType = ShapelessRecipe(
+				NamespacedKey(this, "${it.name.lowercase()}_string_recipe"),
+				ItemStack(Material.STRING, 4)
+			)
 			woolType.addIngredient(1, it)
 			server.addRecipe(woolType)
+
+			val gunpowderrecipe = ShapelessRecipe(NamespacedKey(this, "gunpowderrecipe"), ItemStack(Material.GUNPOWDER))
```

Comment:
gunpowderRecipe

Reply:
gunpowder_recipe

Reply:
fixed

Reply:
I think what he meant was the val should be `gunpowderRecipe` and the namespaced key should be `gunpowder_recipe`, not both as the latter. (Could be wrong lol, idk)

Reply:
Correct.

Reply:
Correct.

Reply:
ah

Reply:
fixed again